ui/app/js/controllers/user_edit_controller.js

```diff
@@ -31,6 +31,15 @@ function ($scope, $modalInstance, CSRF, Permissions, user, users) {
     $scope.loading = false;
   });
 
+  $scope.user.roles = [];
+  Permissions.getUserRoles(user.username)
+  .success(function(response) {
+    $scope.user.roles = response.roles;
+  })
+  .error(function(response) {
+    console.error(response);
```

Comment:
We're trying to get away from silent errors - better to show an error in a modal dialog for this.